### PR TITLE
test: Add WS2022 Test to Pipeline

### PR DESCRIPTION
**Pull request by jaer-tsun**
**Reason for Change**:

Modify existing pipeline to run AKS-E CNI tests on ws2022

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

File: test/apimodels/cniWindows2022.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                        "keyData": ""
+                    }]
+            }
+        },
+        "servicePrincipalProfile": {
+           "clientId": "",
+           "secret": ""
+        }
+    }
+}
```
Comment (rbtr) [inline test/apimodels/cniWindows2022.json:72]: please set your editor to add trailing newlines on save

File: .pipelines/singletenancy/aks-engine/e2e-step-template.yaml [hunk 2]
```diff
@@ -29,10 +29,12 @@ steps:
...
 
       #download source
-      wget https://github.com/Azure/aks-engine/archive/$aksEVersion.tar.gz
+      #wget https://github.com/Azure/aks-engine/archive/$aksEVersion.tar.gz
+      wget https://github.com/jaer-tsun/aks-engine/archive/v1.0.1.tar.gz
 
       #extract source
-      tar -zxf $aksEVersion.tar.gz
+      #tar -zxf $aksEVersion.tar.gz
+      tar -zxf v1.0.1.tar.gz
```
Comment (rbtr) [inline .pipelines/singletenancy/aks-engine/e2e-step-template.yaml:37]: same question?

**Review by rbtr — DISMISSED** [review]

File: .pipelines/singletenancy/aks-engine/e2e-step-template.yaml [hunk 3]
```diff
@@ -29,10 +29,12 @@ steps:
       echo Using AKS-Engine version $aksEVersion
 
       #download source
-      wget https://github.com/Azure/aks-engine/archive/$aksEVersion.tar.gz
+      #wget https://github.com/Azure/aks-engine/archive/$aksEVersion.tar.gz
+      wget https://github.com/jaer-tsun/aks-engine/archive/v1.0.1.tar.gz
```
[the remark this answers is not in the document]
Comment (rbtr) [reply .pipelines/singletenancy/aks-engine/e2e-step-template.yaml:33]: just for until they support ws2022 or forever? if the former, how can we make sure this gets set back to official when that happens?

[the remark this answers is not in the document]
Comment (jaer-tsun) [reply .pipelines/singletenancy/aks-engine/e2e-step-template.yaml:33]: I believe it's forever

File: test/apimodels/cniWindows2022.json [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    "properties": {
+        "orchestratorProfile": {
+            "orchestratorType": "Kubernetes",
+            "orchestratorRelease": "1.23",
+            "orchestratorVersion": "1.23.4",
+            "kubernetesConfig": {
+                "networkPlugin": "azure",
+                "networkPolicy": "azure",
+                "containerRuntime": "containerd",
+                "windowsContainerdURL": "https://github.com/containerd/containerd/releases/download/v1.5.6/containerd-1.5.6-windows-amd64.tar.gz",
```
Comment (matmerr) [inline test/apimodels/cniWindows2022.json:12]: use https://github.com/containerd/containerd/releases/tag/v1.6.2

Comment (jaer-tsun) [reply test/apimodels/cniWindows2022.json]: sure, using v1.6.2 now

**Review by matmerr — APPROVED** [review]
ty for the doing apimodel env dance

**Pull request merged**